tools/who_what_benchmark/requirements.txt

```diff
@@ -8,3 +8,4 @@ pandas>=2.0.3
 numpy>=1.23.5
 tqdm>=4.66.1
 diffusers
+openvino-genai
```

Comment:
it will install public GenAI version, while we want to test latest available (nightly) in precommits.

Reply:
> it will install public GenAI version, while we want to test latest available (nightly) in precommits.

Agree with that but it looks like a problem of CI to care of that. The thing is that WWB requires GenAI while being installed by external users.

Reply:
The problem is that once new API / features / fixes are used in master's version of WWB, they are not available not only on CI, but also for end users (BTW, CI is just one of WWB users). It means WWB is misconfigured. 

Why dependencies of WWB are partially hardcoded on CI? instead of this requirements.txt https://github.com/openvinotoolkit/openvino.genai/blob/e2fa0d002117bbfe6d9a6a4cc413e583c4455131/.github/workflows/llm_bench-python.yml#L104-L107

In requirements for GenAI samples to export models, we have:
https://github.com/openvinotoolkit/openvino.genai/blob/e2fa0d002117bbfe6d9a6a4cc413e583c4455131/samples/export-requirements.txt#L1-L4
which ensures that people who are using latest and greatest samples from master, can convert models and run them via Python or C++ API. I suppose we need something similar for WWB

Reply:
It makes sense. LLMBench also refers to the latest version of GenAI in its [requirements](https://github.com/openvinotoolkit/openvino.genai/blob/e2fa0d002117bbfe6d9a6a4cc413e583c4455131/tools/llm_bench/requirements.txt#L3). @alexsu52, can we do the same and put the nightly version into requirements.txt?